Patches/SQLite3/SQLite3Config.cmake.in

```diff
@@ -0,0 +1,10 @@
+# Config file used to import the SQLite3 build tree.
+set( SQLite3_INCLUDE_DIR "@CMAKE_INSTALL_FULL_INCLUDEDIR@" )
+set( SQLite3_LIBRARY "@CMAKE_INSTALL_FULL_LIBDIR@/sqlite3" )
```

Comment:
Er... I don't think this works? I think you really do want a `find_library` (with appropriate hints) here. (Or, if you really want to avoid that, I believe there are variables for the lib prefix and suffix.)

Reply:
fixed.